gnosis/safe-react
# Catch and log fetching errors

## What it solves
Resolves #2327.

Some network errors weren't caught.

## How this PR fixes it
Adds a try-catch for possible exceptions when fetching the safe info, balances, history, collectibles etc
These errors are just logged to the console to not flood Sentry.

I've also added `Code` in front of the error messages so that it's easier to search for them.

@fernandomg I've deleted `allSettled`, sorry 😅

## How to test it
* Disable network and wait
* It will attempt to fetch the safe info but fail
* Observe the new error messages in the console (see the screenshot below)
* None of the functionality should be affected

## Any extra information
<img width="1094" alt="Screenshot 2021-06-09 at 11 55 36" src="https://user-images.githubusercontent.com/381895/121334474-f95e3c00-c919-11eb-9437-8f5f77bded35.png">
